reV/supply_curve/supply_curve.py

```diff
@@ -251,25 +254,26 @@ def _map_trans_capacity(trans_sc_table):
         nx = trans_sc_table['capacity'] / trans_sc_table['max_cap']
         nx = np.ceil(nx).astype(int)
         trans_sc_table['n_parallel_trans'] = nx
-        mask = nx > 1
 
-        tie_line_cost = trans_sc_table.loc[mask, 'tie_line_cost'] * nx[mask]
+        if (nx > 1).any():
+            mask = nx > 1
```

Comment:
I am a little surprised a mask is needed here since, presumably, `nx[~mask] == 1` (unless capacity = 0, in which case `nx == 0` is still a valid multiplication to apply below).

I will assume there is a subtlety that I am missing here, but I figured I may as well point it out to you

Reply:
No youre right, I think i found an annoying case where the input data was bad (maybe NaN's, i forget) and so i just wanted to be extra careful and not manipulate any data that didnt need manipulating.

Reply:
Ohhhhh, that makes sense. Ok so in that case, it's good to be extra careful!